### PR TITLE
[zh-cn]SYNC pod-lifecycle.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -634,7 +648,7 @@ containers:
 : Indicates whether the application within the container is started.
   All other probes are disabled if a startup probe is provided, until it succeeds.
   If the startup probe fails, the kubelet kills the container, and the container
-  is subjected to its [restart policy](#restart-policy). If a container does not
+ is subjected to its [restart policy](#restart-policy). If a container does not
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:651]:
```suggestion
  is subjected to its [restart policy](#restart-policy). If a container does not
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 2]
```diff
@@ -537,7 +550,7 @@ Each probe must define exactly one of these four mechanisms:
   The target should implement
   [gRPC health checks](https://grpc.io/grpc/core/md_doc_health-checking.html).
   The diagnostic is considered successful if the `status`
-  of the response is `SERVING`.
+  of the response is `SERVING`.
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:553]:
```suggestion
  of the response is `SERVING`. 
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 3]
```diff
@@ -507,6 +507,19 @@ condition to `True` before sandbox creation and network configuration starts.
...
+-->
+### Pod 调度就绪   {#pod-scheduling-readiness-gate}
+
+
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+<!--
+See [Pod Scheduling Readiness](/docs/concepts/scheduling-eviction/pod-scheduling-readiness/) for more information.
+-->
+有关详细信息，请参阅 [Pod 调度就绪](/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness/)。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:521]:
```suggestion
有关详细信息，请参阅 [Pod 调度就绪态](/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness/)。
```

File: content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 4]
```diff
@@ -507,6 +507,19 @@ condition to `True` before sandbox creation and network configuration starts.
 对于没有 Init 容器的 Pod，kubelet 会在创建沙箱和网络配置开始之前将
 `Initialized` 状况设置为 `True`。
 
+<!--
+### Pod scheduling readiness {#pod-scheduling-readiness-gate}
+-->
+### Pod 调度就绪   {#pod-scheduling-readiness-gate}
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/workloads/pods/pod-lifecycle.md:513]:
```suggestion
### Pod 调度就绪态   {#pod-scheduling-readiness-gate}
```

**Review by windsonsea — COMMENTED** [review]
It's a good practice to keep consistent between a link text and its target:

https://kubernetes.io/zh-cn/docs/concepts/scheduling-eviction/pod-scheduling-readiness/

**Comment by tengqm** [comment]
Thanks
/lgtm
/approve

**Pull request merged**